serialized-lob/src/main/java/com/iluwatar/serializedlob/PostGresSerializer.java

```diff
@@ -0,0 +1,91 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.iluwatar.serializedlob;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.iluwatar.serializedlob.PostGresDatabase.connection;
+
+/**
+ *  Creates a single Character Large object(CLOB) from PostgreSQL graph of objects
+ */
+public class PostGresSerializer {
+    public static String graphToLOB(Connection c, String tableName){
+        StringBuilder lob = new StringBuilder();
+        Statement stmt = null;
+        String sql = "\\d+ " + tableName;
+        Map<String,String> columns = new HashMap<>();
+        try{
+            ResultSet columnSets = stmt.executeQuery(sql);
+            String col = "";
+            String type = "";
+
+            while(columnSets.next()){
+                col = columnSets.getString("columns");
+                type = columnSets.getString("Type");
+                columns.put(col,type);
+            }
+            columnSets.close();
+
+            int size = columns.size();
+            lob.append(size);
+            for(String s:columns.keySet()){
+                lob.append(" ").append(s);
+                lob.append(" ").append(columns.get(col));
+            }
+
+        } catch (Exception e){
+            System.err.println( e.getClass().getName()+": "+ e.getMessage() );
+            System.exit(0);
+        }
+
+        sql = "SELECT * FROM " + tableName;
+        try{
+            ResultSet graph = stmt.executeQuery(sql);
+
+            while(graph.next()){
+                for (String s : columns.keySet()){
+                     lob.append(" ").append(graph.getString(s));
+                }
+            }
+
+            graph.close();
+        }catch (Exception e){
+            System.err.println( e.getClass().getName()+": "+ e.getMessage() );
+            System.exit(0);
+        }
+
+        return lob.toString();
+    }
+
+    public static void main(String[] args) {
+    }
```

Comment:
- Use try-with-resources to autoclose resources
- Explain with comments what happens in the code
- System.exit is probably not the correct solution. Maybe just let the method throw the exception?